pkg/cloudprovider/aws/suite_test.go

```diff
@@ -222,6 +222,9 @@ var _ = Describe("Allocation", func() {
 		})
 		It("should launch separate instances for pods with different node selectors", func() {
 			// Setup
+			fakeEC2API.DescribeSubnetsOutput = &ec2.DescribeSubnetsOutput{Subnets: []*ec2.Subnet{
```

Comment:
Shouldn't there simply be a default subnet you can use in this test? I feel it shouldn't be necessary to specify this for this test.

Reply:
Fixed - good catch - turned out to be another bug in fakeEC2API